TVC_MVP/CountyPresnterProtocol.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CountyPresnterProtocol.swift
+//  TVC_MVP
+//
+//  Created by Shane McCully on 01/03/2019.
+//  Copyright © 2019 Shane McCully. All rights reserved.
+//
+
+import Foundation
+
+protocol CountyPresnterProtocol: class {
+
+    init(view: CountyViewProtocol)
+
+    func viewDidLoad()
+
+    func reloadData()
+
+    func numberOfSections() -> Int
+
+    func numberOfRowsInSection() -> Int
+
+    func removeCounty(at indexPath: IndexPath)
+
+    func fetchCounty(for indexPath: IndexPath) -> County?
+
+    func presentAlert(at indexPath: IndexPath)
```

Comment:
The naming of this method misses the point of "The view doesn't know what's going to happen when the user selects the row at the given index path". It's already pointing that it should present an alert.